.github/workflows/library-release.yml

```diff
@@ -1,23 +1,116 @@
-name: Build and Release
+name: Publish c2pa-library binaries
 
 on:
   push:
-    branches: [ "gpeacock/c2pa_c" ]
     tags:
       - 'c2pa-v*' # Trigger on version tags (e.g., v1.0.0)
   workflow_dispatch: # Allow manual triggering of the workflow
 
-env:
-  CARGO_TERM_COLOR: always
-
 jobs:
   build:
+    name: Publish c2pa-library binaries
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ macos-latest, ubuntu-latest, windows-latest ]
```

Comment:
For nightlies, I am all-in for `ubuntu-latest` and `windows-latest`. But for "normal" release builds, I always wonder if we should pin the versions, and update should be a conscious decision.

Reply:
That's a good idea, I'm just not sure what those versions should be. For testing (not in this script) we try nightly and   a minimal version for compatibility. There are lots of questions about what version of what these libraries should support.
I wanted to go with only MUSL builds for linux, but they don't support dynamic libraries, only binary apps.